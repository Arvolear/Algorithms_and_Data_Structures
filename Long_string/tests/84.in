7607465640
246368448999476565252089