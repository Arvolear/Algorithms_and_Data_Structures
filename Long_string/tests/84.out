246368448999484172717729
246368448999468957786449
1874239510543610348140485005721960
32385088629789